xbmc/settings/AdvancedSettings.cpp

```diff
@@ -300,7 +300,6 @@ void CAdvancedSettings::Initialize()
   m_bVideoScannerIgnoreErrors = false;
   m_iVideoLibraryDateAdded = 1; // prefer mtime over ctime and current time
 
-  m_iEpgLingerTime = 60 * 24;           /* keep 24 hours by default */
```

Comment:
I think this answers my previous question, but `iPastDays` is still a bit confusing if we're actually dealing with minutes